<launch>
    <arg name="name" default="robot"/>
    <arg name="map_frame" default="map" />
    <arg name="sync_scan" default="false" />

    <node pkg="pose_publisher" exec="pcl_node" name="pose_to_pcl_$(var name)">
        <param name="map_frame" value="$(var map_frame)" />
        <param name="ignored_ids" value="[$(var name)]" />
        <param name="sync_scan" value="$(var sync_scan)" />

        <remap from="pose" to="/robot_poses" />
        <remap from="pose_cloud" to="/obstacle_cloud" />
        <remap from="scan" to="/scan" />
    </node>

    <!-- <node pkg="pointcloud_to_laserscan" exec="pointcloud_to_laserscan_node" name="pcl2scan">
        <remap from="cloud_in" to="/obstacle_cloud" />
        <remap from="scan" to="/scan_obs" />

        <param name="range_max" value="15.0" />
        <param name="use_inf" value="false" />
    </node> -->
</launch>